nilearn/plotting/img_plotting.py

```diff
@@ -313,6 +314,78 @@ def _load_anat(anat_img=MNI152TEMPLATE, dim=False, black_bg='auto'):
 # Usage-specific functions
 
 
+def plot_4d(img, anat_img=MNI152TEMPLATE, cut_coords=None,
```

Comment:
> > +def plot_4d(img, anat_img=MNI152TEMPLATE, cut_coords=None,

> Yes, I agree. I will change it.

It could be called 'plot_multi_maps', and have a contours option to
switch between contours and continuous colors.